src/services/expenses.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const { Expense } = require('../models/Expense');
+const { Op } = require('sequelize');
+
+const prepareWhere = query => (
+  Object.entries(query).reduce((where, [key, value]) => {
+    switch (key) {
+      case 'from':
+        where.spentAt = {
+          ...where.spentAt,
+          [Op.gt]: value,
+        };
+        break;
+
+      case 'to':
+        where.spentAt = {
+          ...where.spentAt,
+          [Op.lt]: value,
+        };
+        break;
+
+      default:
+        where[key] = value;
+    }
+
+    return where;
+  }, {})
+);
+
+const getAll = query => {
+  const where = prepareWhere(query);
+
+  return Expense.findAll({ where });
+};
+
+const getById = id => Expense.findByPk(id);
+
+const add = data => Expense.create(data);
+
+const remove = async(id) => {
+  await Expense.destroy({
+    where: { id },
+  });
+};
+
+const update = async(id, data) => {
```

Comment:
You can make the `update` function shorter and more readable by returning the promise directly instead of using `async/await`. 🧹